### PR TITLE
Cleaned up my pull request

### PR DESCRIPTION
**Pull request by cecoppinger**
OK nvm I did this all wrong. Ignore this one
edit: i think its fixed now maybe

### CODE REVIEW

**Review by iktani — APPROVED** [review]
works on my end. I'm approving it.

**Review by ryanis-online — CHANGES_REQUESTED** [review]
You still need to rebase this and get rid of the `.vs` directory and all the cached and compiled files

**Review by simpson1986 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1 +1,342 @@
-.DS_Store
\ No newline at end of file
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+
+##We don't need all these I just copied and pasted a template that I found from googling - Clarence
```
Comment (ryanis-online) [inline .gitignore:6]: Good thinking! These templates are sometimes better to use anyways since a lot of times it's easy to forget about these

**Review by ryanis-online — APPROVED** [review]
Very good job!!

**Review by zstephenj — APPROVED** [review]

**Pull request merged**